### ISSUE TITLE
pormo-term nitpicks

### ISSUE THREAD

**Issue by bburdette** [ColorCookie-dev/porsmo#6]
Nice program, saves me the trouble!  

Stuff that would be cool:

- a crate for porsmo-term, so I don't have to clone and `cargo install --path .` from the porsmo-term subfolder
- in the --help:
  - document the abbreviations for the commands - p, s, t.
  - document allowed timer time formats.
- if logging_config.yaml is missing the program dies with vague "file not found".  
  - make that optional?
  - or remove it, I just commented that line out.

FWIW: for nix you can build with 

```
$ cd porsmo-term
$ nix-shell -p cargo alsa-lib pkgconfig
$ cargo install --path .
```

**Comment by ColorCookie-dev** [comment]
There is a cargo install,
```rust
cargo install porsmo
```

about logging, I planned to learn about logging in this project, but got distracted, I agree deleting it would be better, I liked env_logger anyways.

In terms of porsmo-term, I have decided that, I liked the previous version of the project better, I think I will be reverting to that, and publish it. The previous version only contained the terminal version and no gui version.

Originally, I needed p, s, t commands for myself, but didn't think, other people would be interested in using those features. So I didn't document it.

For nix? Could you please elaborate further, I know about nix package manager and shell, what is it, that you're suggesting?

**Comment by ColorCookie-dev** [comment]
@bburdette Oh I hadn't used this program for a while, so I didn't quite remember what you were talking about, when you said document p, s, t. Now that I look at it, I misinterpreted something. I will be adding this to the to do list.

**Comment by bburdette** [comment]
Good to know about the porsmo cargo install.  I like the p.s.t. extra features.  

For nix, that's just what I ended up doing to install it there.  Only of interest to nix users.  Didn't mean that to be a to-do item.  If I use this a lot I may make a nix package at some point.

**Comment by ColorCookie-dev** [comment]
This is now implemented in version 0.3.0 specifically in 66d49e2. The way to specify time now documented in porsmo help through example
